examples/p2p-webrtc/voice-agent/bot.py

```diff
@@ -44,6 +44,7 @@ async def run_bot(webrtc_connection):
             vad_enabled=True,
             vad_analyzer=SileroVADAnalyzer(),
             vad_audio_passthrough=True,
+            audio_out_10ms_chunks=2,
```

Comment:
We don't this parameter with a are using the version from main. 
SmallWebRTC now works with any audio chunk size:
 https://github.com/pipecat-ai/pipecat/blob/main/CHANGELOG.md#fixed